app/helpers/events_helper.rb

```diff
@@ -12,8 +12,8 @@ def event_title_line(event)
   end
 
   def users_for_event(state)
-    req = @event.requests
+    req = @event.requests.includes(:user).select(:user_id).distinct
```

Comment:
I have several points here:

1) We should only use travel sponsorships here
2) We should use accessible_by (like discussed yesterday in IRC!)
3) Please don't query the database and then iterate over the result to get it in the correct format, this can be done directly by the database (you can get the email with a single pluck and don't need to iterate)
4) It is not an r or req, it is a request! This increases the readability and your code quality a lot (I see this a lot in your code...)